tests/opengl/test_config_opengl.py

```diff
@@ -37,35 +37,36 @@ def construct(self):
         self.wait(1)
 
 
-def test_background_color(using_opengl_renderer):
+def test_background_color(config, using_opengl_renderer, dry_run):
     """Test the 'background_color' config option."""
-    with tempconfig({"background_color": WHITE, "verbosity": "ERROR", "dry_run": True}):
-        scene = MyScene()
-        scene.render()
-        frame = scene.renderer.get_frame()
-        np.testing.assert_allclose(frame[0, 0], [255, 255, 255, 255])
+    config.background_color = WHITE
+    config.verbose = "ERROR"
 
+    scene = MyScene()
+    scene.render()
+    frame = scene.renderer.get_frame()
+    np.testing.assert_allclose(frame[0, 0], [255, 255, 255, 255])
 
-def test_digest_file(using_opengl_renderer, tmp_path):
+
+def test_digest_file(config, using_opengl_renderer, tmp_path):
     """Test that a config file can be digested programmatically."""
-    with tempconfig({}):
-        tmp_cfg = tempfile.NamedTemporaryFile("w", dir=tmp_path, delete=False)
-        tmp_cfg.write(
-            """
-            [CLI]
-            media_dir = this_is_my_favorite_path
-            video_dir = {media_dir}/videos
-            frame_height = 10
-            """,
-        )
-        tmp_cfg.close()
-        config.digest_file(tmp_cfg.name)
+    tmp_cfg = tempfile.NamedTemporaryFile("w", dir=tmp_path, delete=False)
+    tmp_cfg.write(
+        """
+        [CLI]
+        media_dir = this_is_my_favorite_path
+        video_dir = {media_dir}/videos
+        frame_height = 10
+        """,
+    )
+    tmp_cfg.close()
+    config.digest_file(tmp_cfg.name)
```

Comment:
It practically doesn't matter because the tests are run in an isolated environment anyways, but what do you think about rewriting this test such that the config file is deleted in the end? The tests could just be moved inside an appropriate context manager. (I can take care of it.)

Reply:
That's probably a good idea, I'm all for cleaning up after our messes :)